templates/migration/init-migrate-hook.yaml

```diff
@@ -72,14 +36,14 @@ spec:
           envFrom:
             - configMapRef:
                 name: {{ include "orchestrate.api.fullname" . }}-hook-envmap
-            {{- if .Values.api.environmentSecrets }}
+            {{- if .Values.migrate.environmentSecrets }}
             - secretRef:
                 name: {{ include "orchestrate.api.fullname" . }}-hook-secretmap
                 optional: true
             {{- end }}
             {{- if .Values.api.existingSecret }}
             - secretRef:
-                name: {{ .Values.api.existingSecret }}
+                name: {{ .Values.migrate.existingSecret }}
```

Comment:
How does one specify a JDBC path/url to use. Should we also have defined credentials defined like we do for blockscout https://github.com/ConsenSys/quorum-kubernetes/blob/master/dev/helm/charts/blockscout/values.yaml#L23